index.js

```diff
@@ -176,13 +190,13 @@ module.exports = function(input, map) {
   // Read the cached information only once and if enable
   if (cache === null) {
     if (config.cache) {
-      var thunk = findCacheDir({
-        name: "eslint-loader",
-        thunk: true,
-        create: true,
-      })
-      cachePath = thunk("data.json") || os.tmpdir() + "/data.json"
       try {
+        var thunk = findCacheDir({
+          name: "eslint-loader",
+          thunk: true,
+          create: true,
+        })
+        cachePath = thunk("data.json")
         cache = require(cachePath)
       }
       catch (e) {
```

Comment:
I presume that we want to read from fallback below? Maybe extract it to `safeReadCache`?